components/admin_console/group_settings/group_settings.jsx

```diff
@@ -14,7 +14,10 @@ import {getSiteURL} from 'utils/url';
 
 export default class GroupSettings extends React.PureComponent {
     static propTypes = {
-        isDisabled: PropTypes.func,
+        isDisabled: PropTypes.oneOfType([
+            PropTypes.func,
+            PropTypes.bool,
+        ]),
```

Comment:
Is this related or something separate?

Reply:
This was a separate issue - isDisabled can be both a function or a bool... there was a console error that showed up on this page because of that